### ISSUE TITLE
Sort 1.3 Terminology (Trac #52)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#52]
It's irritating to try and look up definitions in section 1.3. IMHO, the entries really should be sorted alphabetically, despite the fact that the terms have dependencies on one another.

Migrated from https://trac.ietf.org/ticket/52
```json
{
    "status": "closed",
    "changetime": "2008-11-18T16:53:24",
    "_ts": "1227027204000000",
    "description": "It's irritating to try and look up definitions in section 1.3. IMHO, the entries really should be sorted alphabetically, despite the fact that the terms have dependencies on one another.",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T02:34:22",
    "component": "non-specific",
    "summary": "Sort 1.3 Terminology",
    "priority": "",
    "keywords": "",
    "milestone": "06",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080104T06:10:53: 
* mnot@pobox.com changed component from "" to "non-specific"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080105T03:46:00: fielding@gbiv.com changed milestone from "unassigned" to "02"

**Comment by mnot** [comment]
Trac update at 20080228T05:26:25: mnot@pobox.com changed milestone from "02" to "03"

**Comment by mnot** [comment]
Trac update at 20080717T14:23:05: julian.reschke@gmx.de changed milestone from "03" to "04"

**Comment by mnot** [comment]
Trac update at 20080829T08:45:02: julian.reschke@gmx.de changed milestone from "04" to "unassigned"

**Comment by mnot** [comment]
Trac update at 20081113T23:33:35: mnot@pobox.com changed milestone from "unassigned" to "06"

**Comment by mnot** [comment]
Trac update at 20081118T16:50:33: 
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"
* julian.reschke@gmx.de changed status from "new" to "assigned"

**Comment by mnot** [comment]
Trac update at 20081118T16:53:24: 
* julian.reschke@gmx.de commented:

> Fixed in [413]:
> 
> Resolve #52: Sort Terminology entries (closes #52)


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "assigned" to "closed"
